packages/react/.storybook/Container.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import './polyfills';
 import './_container.scss';
 import { settings } from 'carbon-components';
+import '@carbon/ibmdotcom-styles';
```

Comment:
@annawen1 if we're bringing in the whole styles package, we probably don't need to bring in the individual scss files in the component stories right?